src/remote_interface.h

```diff
@@ -0,0 +1,59 @@
+#ifndef REMOTE_INTERFACE_H_
+#define REMOTE_INTERFACE_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "common.h"
+#include "constants.h"
+#include "remote_common.h"
+#include "uv.h"
+
+#define CONN_MAX 20
+
+typedef struct _pow_remote_context PoW_Remote_Context;
+typedef struct _remote_impl_context RemoteImplContext;
+
+struct _pow_remote_context {
+    /* Thread management */
+    int indexOfContext;
+    /* Arguments of PoW */
+    int8_t input_trytes[TRANSACTION_TRYTES_LENGTH];  /* 2673 */
+    int8_t output_trytes[TRANSACTION_TRYTES_LENGTH]; /* 2673 */
+    int mwm;
+    /* PoW-related information */
+    PoW_Info pow_info;
+};
+
+struct _remote_impl_context {
+    void *context;
+    char *description;
+    /* Connection parameters */
+    amqp_connection_state_t conn[CONN_MAX];
+    /* Thread management */
+    uv_mutex_t lock;
+    bool slots[CONN_MAX]; /* Used to tell which slot is
+                                              available */
```

Comment:
The typesetting of the comment is weird.
Did you use the **clang-format** to check the style?

Reply:
yes